core/node/libp2p/routingopt.go

```diff
@@ -23,6 +24,63 @@ type RoutingOption func(
 	...peer.AddrInfo,
 ) (routing.Routing, error)
 
+// Default HTTP routers used in parallel to DHT when Routing.Type = "auto"
+var defaultHTTPRouters = []string{
+	"https://cid.contact/routing/v1", // https://github.com/ipfs/kubo/issues/9422#issuecomment-1338142084
```

Comment:
```suggestion
	"https://cid.contact/routing/v1", // https://github.com/ipfs/kubo/issues/9422#issuecomment-1338142084
```

(the client adds the `/routing/v1` prefix)